data/templates/libkkc-data/tools/sortlm.py

```diff
@@ -143,13 +144,14 @@ def cmp_header(a, b):
             offset += 1
         unigram_file.close()
 
-        print "writing 2-gram file"
+        print("writing 2-gram file")
         bigram_offsets = {}
         bigram_file = open("%s.2gram" % self.__output_prefix, "wb")
-        keys = self.__ngram_entries[1].keys()
+        keys = list(self.__ngram_entries[1].keys())
```

Comment:
Why is this conversion needed?

Reply:
Because it follows the result of 2to3.
Do you wish not to have list()? and I can delete it.

Reply:
I'm not a python expert but the output of 2to3 is not normative. Instead I would suggest to read [the documentation](https://docs.python.org/3/howto/pyporting.html) and keep the changes as minimal as possible.